src/main/java/com/coreos/jetcd/exception/ClosedSnapshotException.java

```diff
@@ -0,0 +1,19 @@
+package com.coreos.jetcd.exception;
+
+public class ClosedSnapshotException extends EtcdException {
+
+  private static final ClosedSnapshotException DEFAULT_INSTANCE;
+
+  static {
+    DEFAULT_INSTANCE = new ClosedSnapshotException();
```

Comment:
Don't do that  as the stack trace of the exception will be initialized with the location of the first call, an subsequent throw will report the same stack regardless of the location in which you invoke ``throw XyzException.getDefaultInstance()``